File: PART3/ch02/ch02_02/service/api/src/main/java/org/delivery/api/domain/token/ifs/TokenHelperIfs.java
package org.delivery.api.domain.token.ifs;

import org.delivery.api.domain.token.model.TokenDto;

import java.util.Map;

public interface TokenHelperIfs {

    TokenDto issueAccessToken(Map<String, Object> data);
    TokenDto issueRefreshToken(Map<String, Object> data);

    Map<String, Object> validationTokenWithThrow(String token);

}

File: PART3/ch04/ch_03/service/account/src/main/kotlin/org/delivery/account/domain/token/ifs/TokenHelperIfs.kt
package org.delivery.account.domain.token.ifs

import org.delivery.account.domain.token.model.TokenDto

interface TokenHelperIfs {
    fun issueAccessToken(data: Map<String, Any>?): TokenDto?
    fun issueRefreshToken(data: Map<String, Any>?): TokenDto?
    fun validationTokenWithThrow(token: String?): Map<String, Any>?

}